src/BuildingRegistry.Api.BackOffice/Building/BuildingController-Realize.cs

```diff
@@ -46,6 +49,7 @@ public partial class BuildingController
         [SwaggerResponseHeader(StatusCodes.Status202Accepted, "location", "string", "De url van het gebouw.")]
         [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestResponseExamples))]
         [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
+        [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme, Policy = PolicyNames.GeschetstGebouw.DecentraleBijwerker)]
```

Comment:
@ArneD we have multiple policies for buildings. They can be divided into two groups:

geschetst gebouw
ingemeten gebouw.
Not sure which one applies here?

Reply:
Leave it as is and let's ask the business :).